pkg/build/controller/strategy/custom_test.go

```diff
@@ -22,15 +24,21 @@ func TestCustomCreateBuildPod(t *testing.T) {
 
 	expected := mockCustomBuild()
 	actual, err := strategy.CreateBuildPod(expected)
-
 	if err != nil {
 		t.Errorf("Unexpected error: %v", err)
 	}
 
-	if actual.ObjectMeta.Name != expected.PodName {
-		t.Errorf("Expected %s, but got %s!", expected.PodName, actual.ObjectMeta.Name)
+	if e, a := buildutil.GetBuildPodName(expected), actual.ObjectMeta.Name; e != a {
```

Comment:
Yeah, but that limits the amount of test code I need to write, but if you two insists, I'll change that.